src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -58,29 +58,62 @@ export class FilterTimestamp extends HTMLElement {
     );
   }
 
-  onClearFilter = () => {
-    this.startTimeLabel.textContent = 'From Beginning';
-    this.stopTimeLabel.textContent = 'Up to End';
-    this.filterContainer.hidden = true;
+  onClearFilter(clearRequest) {
```

Comment:
Another option would be to make this function to take two boolean parameters, `clearStart` and `clearStop`.

Reply:
I have added boolean parameters now.